lib/puppet/property/list.rb

```diff
@@ -7,19 +7,6 @@ class Property
     #
     class List < Property
 
-      def should_to_s(should_value)
-        #just return the should value
-        should_value
-      end
-
-      def is_to_s(currentvalue)
-        if currentvalue == :absent
-          return _("absent")
-        else
-          return currentvalue.join(delimiter)
```

Comment:
You're right. It doesn't.

Another interesting detail here is that "absent" is subject to translation. Should it be? I don't see any translations of these values in other places.